gio/tests/appmonitor.c

```diff
@@ -5,7 +5,6 @@ static gboolean
 create_app (gpointer data)
 {
   const gchar *path = data;
-  gchar *file;
```

Comment:
This commit went upstream as https://git.gnome.org/browse/glib/commit/?id=f45ec47be1244fd16be339dd5be37af3506e05ca right after 2.48.0 was released - please cherry-pick that one instead.